src/Common/tests/System/Net/HttpTestServers2.cs

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+
+namespace System.Net.Tests
+{
+    internal class HttpTestServers2
```

Comment:
Because we already have an `HttpTestServers` class. :)

It's a point in time thing.  I don't want to disturb the existing `HttpTestServer` class.  Eventually, there will be just an `HttpTestServers` class.

If you can suggest another name other than `HttpTestServers2`, then I can use that.  But looking at .NET naming guidance shows that "2" is ok (esp. for temporary stuff).  Just look at `X509Certificate` and `X509Certificate2`.